api/ceramic.yaml

```diff
@@ -156,9 +156,69 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/ErrorResponse"
+  "/experimental/events/{sep}/{model}":
+    get:
+      summary: Get events matching the interest stored on the node
+      parameters:
+        - name: sep
+          in: path
+          description: name of the separator/sort_key e.g. 'model'
```

Comment:
Name of the field in the Events header that holds the separator value.

Reply:
updated